mayastor/src/bdev/nexus/nexus_noddy_rebuild.rs

```diff
@@ -0,0 +1,90 @@
+//!
+//! This file implements operations for the rebuild process
+//!
+//! `noddy_rebuild` does a noddy rebuild
```

Comment:
There was the question previously about what "noddy" meant. Maybe say "does a simple test rebuild" rather than "does a noddy rebuild"

Reply:
Done